values.yaml

```diff
@@ -120,4 +106,29 @@ relay:
 
   ## Secrets don't like '-'
   secrets:
-    relayCogToken: 
+    relayCogToken:
+
+nginx:
+  name: nginx
+  replicaCount: 1
+  image:
+    repository: nginx
+    tag: 1.11
+  resources:
+    # limits:
+    #   cpu: 100m
+    #   memory: 128Mi
+    requests:
+      cpu: 100m
+      memory: 128Mi
+  serviceType: LoadBalancer
+  ingress:
```

Comment:
Good point. I would argue that since the ingress sits in front if Nginx, it's an Nginx thing. Honestly, unless you need SSL, the Chart doesn't need an ingress anymore... and even that could be dropped into the Nginx config.